src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Accordion.css';
+
+class AccordionItem extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
You don't need to pass props to super unless you want to access `this.props` in the constructor.

Reply:
Offical docs: 

![Uploading Screen Shot 2017-10-31 at 08.59.44.png…]()

Stack Overflow disagreeing: https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e